### ISSUE TITLE
Callout quote component link color is not accessible

### ISSUE THREAD

**Issue by wonilsuhibm** [carbon-design-system/carbon-for-ibm-dotcom#4519]
## Detailed description

The link at the bottom of this component (see the attached image) currently does not pass AA/AAA. It needs to be blue 40.

## Steps to reproduce the issue

1. Open this storybook page: https://ibmdotcom-web-components-staging.mybluemix.net/?path=/story/components-callout-quote--default
2. Check the color of the link

## Additional information

<img width="1303" alt="Screen Shot 2020-11-18 at 6 40 27 PM" src="https://user-images.githubusercontent.com/30183447/99601668-db58da80-29cd-11eb-8417-fef315c62ab7.png">

**Comment by wonilsuhibm** [comment]
<img width="1303" alt="Screen Shot 2020-11-18 at 6 40 27 PM" src="https://user-images.githubusercontent.com/30183447/99601668-db58da80-29cd-11eb-8417-fef315c62ab7.png">
